config/jest.base.config.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  testMatch: [ "**/unit/**/?(*.)+(spec|test).[t]s?(x)" ],
+  testMatch: [ "**/(unit|jest)/**/?(*.)+(spec|test).[t]s?(x)" ],
```

Comment:
unit tests will now be valid under a unit or jest directory. 
The intention is the jest directory is the new location for jest true unit tests.